### ISSUE TITLE
PATCH: add Epub export to Org layer

### ISSUE THREAD

**Issue by tidux** [syl20bnr/spacemacs#10846]
This patch to layers/+emacs/org/packages.el is tested and working.

```
30a31
>     ox-epub
525a527,529
> 
> (defun org/init-ox-epub ()
>   (spacemacs|use-package-add-hook org :post-config (require 'ox-epub)))
```

I can resubmit this as a PR if requested.

**Comment by robbyoconnor** [comment]
Submit as a pull request...against `develop`

**Comment by syl20bnr** [comment]
Anyone wants to submit a PR for this patch ?

**Comment by TinySong** [comment]
should we add some shortcut  for org-epub-export-to-epub ?

**Comment by duianto** [comment]
The PR:
Fixes #10846, added epub support for Org layer. #11926
Has been cherry-picked into the develop branch.

**Comment by duianto** [comment]
Hopefully it worked this time 😄 
Thank you for contributing to Spacemacs!
The PR has been cherry-picked into the `develop` branch, you can safely delete your branch.
